bundle/edu.gemini.spModel.core/src/main/scala/edu/gemini/spModel/core/HorizonsDesignation.scala

```diff
@@ -0,0 +1,25 @@
+package edu.gemini.spModel.core
+
+import scalaz._, Scalaz._
+
+/** 
+ * Unique Horizons designation, which must be formatted differently depending on the object type.
+ * The `queryString` should yield a unique result.
+ */
+sealed abstract class HorizonsDesignation(val designation: String, val queryString: String) 
+  extends Product with Serializable
+
+object HorizonsDesignation {
+
+  final case class Comet(des: String)     extends HorizonsDesignation(des, s"DES=$des;CAP")
```

Comment:
Thanks. Probably this is not documented anywhere in the Horizons documentation. So probably it's worth some words in this code or somewhere else?